module/index.js

```diff
@@ -106,6 +106,7 @@ import toLowerCase from './toLowerCase';
 import toUpperCase from './toUpperCase';
 import uncurry from './uncurry';
 import uncurry3 from './uncurry3';
+import unfold from './unfold';
```

Comment:
yeah it seems like in order to allow recursion to work within `1-liners` we would have to make everything a named export so that users as you said wouldn't get confused 
otherwise we have to make it use the  `ƒ(...){return...}` style